app/views/home/index.html.slim

```diff
@@ -1,2 +1,12 @@
-h1 Home#index
-p Find me in app/views/home/index.html.erb
+h1 Welcome to the Home page
+p Your awesome!
+
+- if user_signed_in?
+  p #{current_user.email}
+  = button_to 'Logout', destroy_user_session_path, method: :delete
+
+- else
+  = link_to 'Login', new_user_session_path,
+    class: 'button button--secondary'
+  = link_to 'Sign up', new_user_registration_path,
+    class: 'button button--primary'
```

Comment:
```suggestion
  = link_to 'Sign up', new_user_registration_path
```

Reply:
I think we can omit this class from this PR 🥹
Because this PR is about backend ticket, right 🙌